src/alpaka/CondFormats/SiPixelFedCablingMapGPUWrapper.h

```diff
@@ -0,0 +1,31 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_SiPixelFedCablingMapGPUWrapper_h
+#define RecoLocalTracker_SiPixelClusterizer_SiPixelFedCablingMapGPUWrapper_h
+
+#include "CondFormats/SiPixelFedCablingMapGPU.h"
+
+#include "AlpakaCore/alpakaCommon.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class SiPixelFedCablingMapGPUWrapper {
+  public:
+    using CablingMapDeviceBuf = AlpakaDeviceBuf<SiPixelFedCablingMapGPU>;
+
+    explicit SiPixelFedCablingMapGPUWrapper(CablingMapDeviceBuf cablingMap, bool quality)
+        : cablingMapDevice_{std::move(cablingMap)}, hasQuality_{quality} {}
+    ~SiPixelFedCablingMapGPUWrapper() = default;
+
+    bool hasQuality() const { return hasQuality_; }
+
+    //const CablingMapDeviceBuf& cablingMap() const { return cablingMapDevice_; }
+    // TO DO: what about returning a view only?
```

Comment:
Could you elaborate this TODO comment?

Reply:
At first I had a version which was returning a DeviceBuf. I didnt like it, as the client code did not need to own the data, so I just changed it to 
`const SiPixelFedCablingMapGPU* cablingMap() const { return alpaka::getPtrNative(cablingMapDevice_); }`
Which return a non-owning, raw pointer.

i just forgot to remove the to do comment.